executors/sshexec/block_volume.go

```diff
@@ -42,7 +42,7 @@ func (s *SshExecutor) BlockVolumeCreate(host string,
 		auth_set = "disable"
 	}
 
-	cmd := fmt.Sprintf("gluster-block create %v/%v  ha %v auth %v  %v %vG --json",
+	cmd := fmt.Sprintf("gluster-block create %v/%v  prealloc full ha %v auth %v  %v %vG --json",
```

Comment:
@obnoxxx  @raghavendra-talur this `prealloc full` has broken the command line.  Have you tested it ?

Reply:
@obnoxxx jfyi  ^^^^